src/badgehub-api-client/index.ts

```diff
@@ -6,9 +6,31 @@ interface BadgehubApp {
   slug: string;
   category_slug: string;
   user_name: string;
+  devices: BadgehubDevice[];
 }
 
-export async function getApps() {
+export async function getApps(category: string, device: string) {
```

Comment:
We're treating these parameters as optional underneath - which means we're actually (according to typescript) looking for empty string, not undefined, when the category/device isn't specified. I'd prefer to use undefined rather than empty string to represent "any" here.

```suggestion
export async function getApps(category?: string, device?: string) {
```

Further, I'd suggest merging this with `getAllApps` - having one method to get apps that optionally takes filter parameters is a more established pattern in my experience. I'd suggest the following signature:

```suggestion
export async function getApps(options?: {category?: string, device?: string}) {
```